src/windows_stubs.cr

```diff
@@ -73,3 +73,11 @@ end
 def sleep(time : Time::Span)
   LibC.Sleep(time.total_milliseconds)
 end
+
+class Crystal::Thread
```

Comment:
Perhaps move this to `src/crystal/system/win32/thread.cr`

Reply:
IIRC you previously asked to have this stub in `windows_stubs` :D

Reply:
haha, well, oops

I think that was when everything was ifdefs?

Either way, I think i've changed my mind since the rest of `Thread` is a stub too. What do you think? It's not particularly important either way.

Reply:
I agree, let's prepare for the win32 implementations by having stubs defined at their expected place, with a TODO: implement :-)